config/samples/v1_prefectworkpool_external_prefect_cloud.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
dumb q, are these `samples/` used anywhere for the e2e tests? or are these just manifests for manual testing?

Reply:
Oh great q I don't know about that, hmmm

Reply:
Yeah don't think they're used anywhere, just there as examples for end users: https://book.kubebuilder.io/getting-started.html?highlight=samples#sample-of-custom-resources

Reply:
They'd probably be helpful in CI when we want to do a full end-to-end test against a real instance, could `kubectl apply` these and confirm the resources come up sucessfully. Would also ensure our examples are functional